application/main.c

```diff
@@ -1,4 +1,37 @@
+/* Copyright Ville Hiltunen 2018 <hiltunenvillej@gmail.com>
+ *
+ * All code here is open source
+ */
+#include <stdio.h>
+#include <string.h>
 
-int main(int ac, char **av) {
+#include "thread.h"
+#include "leds.h"
+#include "color.h"
+#include "xtimer.h"
+
+
+int main(int ac, char **av)
+{
+    //Initialize LED API. This starts a new thread.
+    leds_init();
+    LED2_TOGGLE;
+    color_rgba_t led_color = {
+        .color = { .r = 255, .g = 0, .b = 0},
+        .alpha = 100
+    };
+    //Change the color at start to red.
+    leds_set_color(led_color);
+    //Slowly increase blueness
+    uint8_t b = 0;
+    for (;;){
+        xtimer_sleep(1);
+        b = b + 10;
+        if (b > 240){
+            b = 0;
+        }
+        led_color.color.b = b;
+        leds_set_color(led_color);
+    }
```

Comment:
Since you have an infinite loop, you should not have a `return` but `/* NOTREACHED */`.